test/integration/BatchTest.py

```diff
@@ -101,14 +101,33 @@ def test_delete(self):
         recipient_id = response.id
 
         # Create Batch
-        payload = {"payments": [{"recipient": {
-            "id": recipient_id}, "sourceAmount": "65", "memo": "", "sourceCurrency": "EUR"}]}
+        payload = {
+            "payments": [
+                {
+                    "recipient": {
+                        "id": recipient_id
+                    }, 
+                    "sourceAmount": "65", 
+                    "memo": "", 
+                    "sourceCurrency": "EUR"
+                }
+            ]
+        }
         response = self.client.batch.create(payload)
         batch_id = response.id
 
         response = self.client.batch.delete(batch_id)
         self.assertTrue(response)
 
+        # Test - Delete Multiple
+        batch1 = self.client.batch.create(payload)
+        batch2 = self.client.batch.create(payload)
+        response = self.client.batch.delete_multiple({
+            "ids":[
+                batch1.id,
+                batch2.id
+        ]})
+
```

Comment:
Is there any assert to check whether the delete multiple is success or not?

Reply:
We could try finding deleted batch ID which would result in a HTTP 404, but whenever any non-200 response is returned, the SDK throws an exception.
I think I can catch that exception and assert if the resources were `not_found`.
Would that be okay?

Reply:
sounds good